far2l/CMakeLists.txt

```diff
@@ -173,6 +173,11 @@ else()
     SET (WINPORT -Wl,--whole-archive WinPort -Wl,--no-whole-archive)
 endif()
 
```

Comment:
i think below lines are redundant, add_dependencies must be enough

Reply:
Unfortunately no,
You can try by yourself. I've just  reproduced this again, by commenting out **set_source_file_properties** section below. Main file doesn't recompile.
I'm not familiar with Cmake internals, just found similar issue:
https://cmake.org/pipermail/cmake/2014-October/058791.html
That was why I implemented fix this way